index.html

```diff
@@ -6119,32 +6123,39 @@ <h2>VR/AR Virtual Guide (for gaming, medical purposes, etc.)</h2>
         <dt>Motivation</dt>
         <dd>
 
-          Users can be guided by a virtual assistant through an area of interest with AR/VR to visualise events or
-          structures and an annotated map or space to provide additional geospatial guidance in sync, including
-          locations of other WoT devices. Two synchronized displays can offer greater insight and provide clearer
-          guidance to the user by showing different views of the same location, e.g. a map.
+          Using a wearable semi-transparent display,
+          users can be guided by a virtual assistant through a physical area of interest with 
+          a rendered overlay to visualize events, annotate structures and other physical features, 
+          or visualize live and historical data associated with features of interest (which may or
+          may not be at the same physical location as the sensor generating the data).
+          An annotated map may provide additional geospatial guidance, including
+          identification of landmarks, locations of devices.   The system may also guide the 
+          user along a specific trajectory.
 
         </dd>
         <dt>Expected Devices</dt>
         <dd>
 
           <ul>
-            <li>Display (possibly a VR Goggle) / speakers for output</li>
-            <li>3D camera / motion sensor for input (possibly microphone for speech recognition)</li>
-            <li>other various devices as output actuators and/or input sensors</li>
+            <li>Wearable, semi-transparent head-mounted display</li> 
+            <li>Headphones for speakers for audio output</li>
+            <li>Geopose and motion estimator (various technologies can be used)</li>
```

Comment:
The term "Geopose" should be explained.

Reply:
Could simply change that to "Location and orientation".  I would say just changing it to "pose" but not everyone knows what that means either ;)